### DISCUSSION TITLE
chatgpt support

### DISCUSSION THREAD

**Discussion by yin0713** [apernet/hysteria#1223] [General]
when I use this proxy in HK to connect to www.chatgpt.com.
I was detected by it that I am a proxy.
how to do about it?

**Comment by tobyxdd** [comment]
1. Their detection is based on IP address and has nothing to do with what protocol you are using
2. ChatGPT doesn't allow users from Hong Kong in the first place

**Comment by iopq** [comment]
just connect to warp outside the proxy and it will work
